modules/node/handler/node_form_response.e

```diff
@@ -228,6 +228,12 @@ feature -- Form
 			ts.set_default_value ("Preview")
 			f.extend (ts)
 
+			if a_node /= Void and then a_node.id > 0 and then has_permission ("delete " + a_name) then
+				create ts.make ("op")
+				ts.set_default_value ("Delete")
```

Comment:
note for later, for now, we don't do anything for i18n, but at some point, we'll need to find solution, and probably we'lll have to do

ts.set_default_value ("Delete")
with

ts.set_default_value (i18n ("Delete"))
i18n or other name such as "translated" or "translation"